docs/reference/user/activation.md

```diff
@@ -0,0 +1,150 @@
+# Activation {#RefActivation}
+
+_Author: Artyom Kazak_
+
+---
+
+A user is called _activated_ they have a verified identity -- e.g. a phone number that has been verified via a text message, or an email address that has been verified by sending an activation code to it.
+
+A user that has been provisioned via single sign-on is always considered to be activated.
+
+## Activated vs. non-activated users {#RefActivationBenefits}
+
+Most functionality is available to all users, but non-activated users can not [connect](connection.md) to others, nor can connection requests be made to anonymous accounts from verified accounts. As the result,
+
+* A non-activated user cannot add other users to conversations. The only way to participate in a conversation is to either create a new conversation with link access or to use a link provided by another user.
+
+* A non-activated user cannot get auto-connected to others as a result of an address book upload.
+
+* A non-activated user can only ever see the public profiles of other users, i.e. no email addresses, phone numbers or other personal information.
+
+## API {#RefActivationApi}
+
+### Requesting an activation code {#RefActivationRequest}
+
+During the [standard registration flow](registration.md#RefRegistrationStandard), an activation code will be sent to provided email/phone. However, it is also possible to request an activation code _before_ registering an account:
```

Comment:
What is the purpose or implication of requesting one before registering an account? Is it worth mentioning this behaviour here at all? Maybe better to leave it as 'undefined' :P

Reply:
This paragraph is incorrect (outdated). These days, the second flow (requesting an activation code _before_ registering an account) is what all clients (web, android, iOS) do.

1) /activate/send
2) /register using code received from 1)

Please adapt the description to make this clearer, rather than, as Chris suggests, not mentioning this.

Reply:
> These days, the second flow (requesting an activation code before registering an account) is what all clients (web, android, iOS) do

Is there a good reason why we never got rid of the other flow?

Reply:
> Please adapt the description to make this clearer

Done.

Reply:
> Is there a good reason why we never got rid of the other flow?

Backwards compatibility with older clients. Possibility of changing flows again in the future.